modules/ROOT/pages/reliability-patterns.adoc

```diff
@@ -27,57 +27,38 @@ See xref:transaction-management.adoc[Transaction Management] for more informatio
 
 If you have a web application that uses a nontransactional connector, such as HTTP, follow a reliability pattern to ensure reliable messaging for your application.
 
-[NOTE]
-VM file persistency is disabled on clusters, so VM endpoints persist in-memory in a clustered topology.
-
 == Comparing Endpoints in Reliability Patterns
 
 You can couple the reliable acquisition flow with any type of transactional endpoint. You don't have to use a VM endpoint in a reliability pattern. The following diagram illustrates a reliability pattern where the reliable acquisition flow delivers a message to a JMS endpoint:
 
 image::reliabilitypatternwithjms.png[ReliabilityPatternwithJMS]
 
-The VM connector uses in-memory queues by default, this makes it much faster than other transactional endpoints such as JMS.
-
 The following table lists the advantages and disadvantages of using VM, JMS, and JDBC endpoints in a reliable acquisition flow.
 
 [%header,cols="10a,60a,30a"]
 |===
 |Endpoint |Advantage |Disadvantage
-|VM (standalone Mule)
-|
-* Faster than JMS because everything is in memory and in-process.
-* Provided ready to use with Mule. There is no need for additional software.
-|
-* Data is lost when Mule exits.
-* Non-clustered.
 |File-based VM (standalone Mule)
 |
 * Provided ready to use with Mule. There is no need for additional software.
 |
-* Non-clustered.
-|VM (Mule HA cluster)
-|
-* Faster than JMS because everything is in shared memory.
-* Provided ready to use with Mule. There is no need for additional software.
-|
-* Data is lost when the entire Mule cluster exits.
+* Only applies to non-clustered environments.
 |JDBC
 |
 * Depending on resources available, JDBC can hold a large amount of information.
 |
+* Requires additional software (database server).
 * Can be complex to use since proper DBMS schemas are required.
-|JMS (not enabled for persistence)
-|
-* Allows sharing data with other Mule instances or non-Mule clients.
-|
-* It is slower than VM because it is in another process.
 |JMS (enabled for persistence)
 |
-* It is the most reliable choice because messages are persisted to disk.
+* Applies to both clustered and non-clustered environments.
 |
-* It is slower than VM because of disk access.
+* Requires additional software (message queue server).
 |===
 
+[NOTE]
+In memory persistence setups for both VM and JMS are excluded from the comparison since they can lose messages; therefore, they are not considered reliable.
```

Comment:
```suggestion
The comparison doesn't include in-memory persistence setups for VM and JMS because they can lose messages and are not considered reliable.
```